extensions/apps/extensions/src/components/pages/info-page/info-page.tsx

```diff
@@ -119,6 +119,13 @@ export const InfoPage: React.FC<InfoPageProps> = ({ appId }) => {
     }
   }, [appData?.applicationType, t]);
 
+  const coverImageSrc = useMemo(() => {
```

Comment:
can you please add some comments here explaining the usecase? Why it returns `/public?`

Reply:
that is something that i still need to fix, I need to add the resources to the ds-core first